src/plugin.js

```diff
@@ -190,6 +196,7 @@ const onPlayerReady = (player, options) => {
         <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
           <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
           ${details}
+          <i><span id="fmsg" style="float:right;font-size:75%">${flashMessage}</span></i>
```

Comment:
Ok I see. I think for multiple players on  page, the `id `attribute will make things harder.
Also shouldn't the CSS for `span` element be better inline instead of within the `plugin.scss` file ?